src/Dependencies/Composition/Composition.csproj

```diff
@@ -1,13 +1,15 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\..\..\build\Targets\ProducesNoOutput.Settings.targets" />
   <PropertyGroup>
     <ProjectGuid>{A57DDFE5-AB0E-4371-98E5-11B9218DF11C}</ProjectGuid>
-    <ProjectTypeGuids>{786C830F-07A1-408B-BD7F-6EE04809D6DB};{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
+    <ProjectTypeGuids>{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}</ProjectTypeGuids>
     <ProjectSystemLayer>Dependency</ProjectSystemLayer>
+    <TargetFramework>netstandard1.0</TargetFramework>
+    <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wpa81+wp8</PackageTargetFallback>
```

Comment:
That is the "short" name :) We don't always need this for netstandard projects and only need this if there are some libraries like Microsoft.Composition being imported. We really should move to System.Composition which gets rid of the need for this but i'll do that in a follow up change. Didn't want to change semantics in this PR.